### PR TITLE
[RP] Scala and Python ported code.

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,28 @@ Notes:
 ---
 
 Follow me on Twitter to get updates about the book and Apache Spark: [@jgperrin](https://twitter.com/jgperrin). Join the book's community on [Facebook](https://www.facebook.com/SparkWithJava/) or in [Manning's community site](https://forums.manning.com/forums/spark-in-action-second-edition?a_aid=jgp).
+
+
+
+It explains how to run Pyspark/Python application
```
Comment (jgperrin) [inline README.md]: Check chapter 1's readme to see how I did it...

File: README.md [hunk 2]
```diff
@@ -10,3 +10,28 @@ Notes:
...
+
+    1. Clone this project
+       Assume that cloned this project to ${MY_HOME_DIR}
+
+    2. cd ${MY_HOME_DIR}/src/main/scala
+
+    3. Execute the following spark-submit command to create a jar file to our this application
+    ```
+    spark-submit net.jgp.books.spark.ch02.lab100_csv_to_db.Csv2RelationalDatabaseApp
```
Comment (jgperrin) [inline README.md]: No need to compile?

File: pom.xml [hunk 3]
```diff
@@ -93,6 +98,27 @@
           </arguments>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>net.alchim31.maven</groupId>
+        <artifactId>scala-maven-plugin</artifactId>
+        <executions>
+          <execution>
```
Comment (jgperrin) [inline pom.xml:109]: This fails on my computer

File: pom.xml [hunk 4]
```diff
@@ -18,6 +18,11 @@
   </properties>
 
   <dependencies>
+    <dependency>
```
Comment (jgperrin) [inline pom.xml:23]: Needed? If so explain and say it's only for Scala

File: pom.xml [hunk 5]
```diff
@@ -93,6 +98,27 @@
           </arguments>
         </configuration>
       </plugin>
+      <plugin>
```
Comment (jgperrin) [inline pom.xml:104]: Why?

**Review by jgperrin — CHANGES_REQUESTED** [review]
Look at my comments. Check the necessity of all those python files.

File: pom.xml (hunk 4 above)
Comment (rambabu-posa) [reply pom.xml:23]: Where to explain? I think you're expecting a comment in pom.xml file, will do.

File: pom.xml (hunk 5 above)
Comment (rambabu-posa) [reply pom.xml:104]: To enable scala in a java project

File: pom.xml (hunk 3 above)
Comment (rambabu-posa) [reply pom.xml:109]: I didnt get any error. send me error stack

File: README.md (hunk 2 above)
Comment (rambabu-posa) [reply README.md]: No need just to run this program. If required, we can this kind of steps any.
I didnt find any steps for Spark/Java

File: pom.xml [hunk 6]
```diff
@@ -98,26 +100,21 @@
...
           </execution>
-          <execution>
-            <id>scala-test-compile</id>
-            <phase>process-test-resources</phase>
-            <goals>
-              <goal>testCompile</goal>
-            </goals>
-          </execution>
         </executions>
+        <configuration>
```
Comment (jgperrin) [inline pom.xml:115]: Why?

File: pom.xml (hunk 5 above)
Comment (jgperrin) [reply pom.xml:104]: Please use sbt

File: README.md [hunk 7]
```diff
@@ -24,14 +16,24 @@ It explains how to run Pyspark/Python application
...
     spark-submit net/jgp/books/spark/ch02/lab100_csv_to_db/csvToRelationalDatabaseApp.py
     ```
-It explains how to run Pyspark/Python application
+## Running Scala
 
     1. Clone this project
        Assume that cloned this project to ${MY_HOME_DIR}
 
-    2. cd ${MY_HOME_DIR}/src/main/scala
+    2. cd ${MY_HOME_DIR}
```
Comment (jgperrin) [inline README.md:24]: Not matching the format we agreed on for chapter 1

**Review by jgperrin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**